api/swagger.yaml

```diff
@@ -2102,6 +2102,16 @@ definitions:
                     SpreadDescriptor:
                       description: "label descriptor, such as engine.labels.az"
                       type: "string"
+          Platforms:
+            description: "An array of supported platforms."
+            type: "array"
+            items:
+              type: "object"
+              properties:
+                Architecture:
```

Comment:
@stevvooe @nishanttotla we can make those changes in 17.07 without causing backward compatibility issues?